src/AccessibilityInsights.Extensions.GitHub/PlaceHolderTextBox.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+
+namespace AccessibilityInsights.Extensions.GitHub
+{
+    public class PlaceHolderTextBox : TextBox
+    {
+        public string PlaceHolder { get; }
+        public SolidColorBrush BlackBrush { get; }
+        public SolidColorBrush GrayBrush { get; }
+
+        public PlaceHolderTextBox()
+        {
+            PlaceHolder = Properties.Resources.PlaceHolder;
+            BlackBrush = Application.Current.Resources["TextBrush"] as SolidColorBrush;
```

Comment:
Will this be responsive to color theme changes that occur while the TextBox is displayed? It feels like it would retain the pre-change colors ☹️

Reply:
it doesn't, we already have this issue in other places and we still didn't fix it, I'm not sure if we can do something about that